packages/paste-website/src/pages/customization/customizing-component-elements.mdx

```diff
@@ -33,7 +33,179 @@ export const getStaticProps = async () => {
 
 <content>
 
-We are currently working on this feature and providing support for it gradually across the entire suite of components. Watch this space!
+As a designer or developer of a Twilio Programmable App, you may need to customize a component to match your specific design requirements. Use Paste’s Customization Provider API to customize individual components or create entirely new ones based on existing components.
+
+The Customization Provider seamlessly integrates with the current theme so that your customizations work perfectly with the default or dark base themes, or any fully custom theme you provide. To make that as easy as possible for you, we have taken the decision to provide some guardrails with that API, only allowing design token values on certain CSS properties where it makes sense.
```

Comment:
suggestion: link to themes page

Reply:
I can't necessarily link the general theme page as customization provider doesn't have access to the same themes. I can link to the creating a custom theme page though